<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceWidget</class>
 <widget class="QFrame" name="ServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>95</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="BtnControl">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>8</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConf">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>8</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LName">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LProgramFile">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>35</y>
     <width>425</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LConfigFile">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>62</y>
     <width>425</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LWorkDir">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>8</y>
     <width>425</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
